### ISSUE TITLE
Pagination click issue

### ISSUE THREAD

**Issue by MUYUSY** [zefoy/ngx-swiper-wrapper#209]
config: SwiperConfigInterface = {
    // a11y: true,
    direction: 'vertical',
    slidesPerView: 1,
    keyboard: false,
    mousewheel: true,
    scrollbar: false,
    navigation: false,
    pagination: {
      el: '.swiper-pagination',
      type: 'bullets',
      clickable: true,
      hideOnClick: false,
    },
  };

There are five div with different background image and no any other content to swipe, everything works fine but only when click the pagination buttons, sometimes it goes to wrong image, such as when I click the fourth page, the swiper shows the second images.

Did anyone know how to fix this?

**Comment by MUYUSY** [comment]
I added a click listener in the page such as : 
this.renderer.listen('document', 'click', (event) => {
      console.log(event.target);
}
and write the onIndexChange function:
onIndexChange (index: number) {
    console.log('goto index:' + index);
  }

the log shows that:
![image](https://user-images.githubusercontent.com/16941350/53611524-e2dcbc80-3c09-11e9-855d-2e90da7d43ee.png)
the last 2 logs, there is only onIndexChange event but no click event, but actually I click the third pagination icon.

further I click the image on the page, I found that sometime click the image will change the index, like this:
![image](https://user-images.githubusercontent.com/16941350/53611665-5bdc1400-3c0a-11e9-8e6b-020344a085a3.png)

Does anyone knows what is the reason and how to solve it?

**Comment by sconix** [comment]
What you are describing does not make much sense and its not happening in the example app. So maybe you could try to reproduce your issue in Stackblitz. With that little information there is nothing shown that could explain the behavior you are seeing. So would need to see the whole code and preferably in Stackblitz.

**Comment by MUYUSY** [comment]
https://stackblitz.com/edit/angular-utp6tv?file=src%2Fapp%2Fapp.component.ts
 @sconix above is the Stackblitz, the same behavior happened. when you click the icon of the pagination, it will goes to the wrong slide.

**Comment by MUYUSY** [comment]
@sconix  OK, I think I find the root course, I did not set the threshold so it was the default value which was 0. After set the threshold to 50 as the example, everything works fine. 

At first I thought it was because the 'noSwiping' config, and I set 
noSwiping: true,
noSwipingClass: 'swiper-slide', 
This avoided the behavior of click the image which change the image. 

Anyway, thanks very much for answer me such soon.
